lab-beeksma/lib/dataFromFile.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+
+function logSpecificFile(filename, cb) {
+  fs.readFile(`${__dirname}/../assets/${filename}`, function (err, res) {
+    if(err){
+      cb(err,undefined);
```

Comment:
`undefined` here is redundant; JS handles that automatically for unspecified arguments.

More importantly, though, you need to `return` here (either `return cb(err)` or as a separate statement) to prevent a second `cb` call.